### ISSUE TITLE
Bug for SegNet Maybe

### ISSUE THREAD

**Issue by holmosaint** [trypag/pytorch-unet-segnet#2]
Hi:) It seems that there are 2 bugs in your SegNet code. Both of them lie in the _Decoder class in function "__init__". The out channels of conv2d and batchnorm2d should be 'n_out_feat' rather than 'n_in_feat'.

**Comment by trypag** [comment]
Nice catch
